src/app/editor-toolbar-container/editor-toolbar-container.component.ts

```diff
@@ -1,13 +1,15 @@
-import { Component, Output, EventEmitter } from '@angular/core';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
 
 @Component({
   selector: 'me-editor-toolbar-container',
   templateUrl: './editor-toolbar-container.component.html',
   styleUrls: ['./editor-toolbar-container.component.scss']
 })
 export class EditorToolbarContainerComponent {
```

Comment:
why this is called container? (I know that this is not part of the PR)

Reply:
@zzacharo can you check if this should change?

Reply:
Because it contains `Save` and `Search` component.

Reply:
`Toolbar` would be more simple.